content/applications/sales/sales/send_quotations/orders_and_variants.rst

```diff
@@ -1,78 +1,116 @@
+
+Once a selection is made, click :guilabel:`Save` on the product form to apply the setting.
 
 .. image:: orders_and_variants/sales-variant-selection.png
    :align: center
    :alt: Sales variant selection.
 
-.. note::
-   The product must have at least two variants (one attribute with two values) created in order for
-   the :guilabel:`Sales Variant Selection` option to appear. Click :guilabel:`Add a line` under
-   :guilabel:`Attribute` to add attributes and values to this product and create product variants.
+Product configurator
+====================
 
-Use the Product Configurator
-============================
-
-The :guilabel:`Product Configurator` appears on a quotation or sales order when products are added
-that have both variants configured and the :guilabel:`Product Configurator` option selected.
+The *Product Configurator* appears on a quotation or sales order when products that have **both**
+variants configured are added, and the :guilabel:`Product Configurator` option is selected on the
+product form.
 
 The :guilabel:`Product Configurator` lets salespeople choose exactly which product variant to add
-to the quotation or sales order using a format similar to online shopping. Color options display
-any HTML color codes set up in the :guilabel:`Color` attribute. Features with an additional cost
-will display the additional cost next to their name. Once the desired product variant attributes
-are selected, click :guilabel:`Add` to add the product variant to the quotation or sales order.
+to the quotation or sales order, using a format similar to online shopping. Color options display
+any HTML color codes set up in the :guilabel:`Color` attribute.
+
+Features with an additional cost display the additional cost next to their name. Once the desired
+product variant attributes are selected, click :guilabel:`Add` to add the product variant to the
+quotation or sales order.
 
 .. image:: orders_and_variants/configurator-price-extras.png
    :align: center
    :alt: Product Configurator and Price Extras.
 
 By default, the product variant attributes are displayed on the quotation or sales order in the
-:guilabel:`Description` column, but the :guilabel:`Product Variant` column can be added through the
-additional options menu (3 vertical dots icon).
+:guilabel:`Description` column, but a dedicated :guilabel:`Product Variant` column can be added,
+via the additional options menu (represented by the 3 vertical dots icon, on the far-right side of
+the columns, in the :guilabel:`Order Lines` tab).
+
+.. image:: orders_and_variants/product-variant-column-option.png
+   :align: center
+   :alt: Product variant column option in order lines tab of quotation.
+
+Just click the checkbox next to :guilabel:`Product Variant`, and that column appears in the
+:guilabel:`Order Lines` tab.
+
+Variant grid entries
+====================
+
+*Variant Grid Entries* appear on a quotation or sales order when products that have both
```

Comment:
No need for caps here since it is not a proper noun
```suggestion
*Variant grid entries* appear on a quotation or sales order when products that have both variants
```